com.wps.WPS.json

```diff
@@ -0,0 +1,94 @@
+{
+    "app-id": "com.wps.WPS",
+    "branch": "alpha",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "tags": ["proprietary"],  
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=host",
```

Comment:
Isn't `home` enough for most use cases?

Reply:
Yes, though I am following LibreOffice, which also speciflied --filesystem=host.

Reply:
Once you have `home` it is minimal to jump to `host` and will save the complaint when somebody tries to access a file outside of home like a mounted drive.

Reply:
Why even not `--filesystem=xdg-documents`?